k8s/k8s.go

```diff
@@ -191,7 +201,10 @@ func validateCvesData(cves []*osv.OSV) error {
 					}
 				}
 				for _, r := range v.Ranges {
-					for i := 1; i < len(r.Events); i++ {
+					if len(r.Events)%2 != 0 {
```

Comment:
The events aren't necessarily a pair in the OSV spec.

```
"ranges": [ {
    "type": "SEMVER",
    "events": [
      { "introduced": "0" },
    ]
} ]
```

cf. https://ossf.github.io/osv-schema/#unfixed-vulnerability-example

But it's okay to have this validation if it is a requirement of our script. @chen-keinan Do you think our k8s advisories must be a pair?

Reply:
I can remove this validation as anyway we are doing PR review manually